rpcserver.go

```diff
@@ -49,6 +49,14 @@ import (
 	"github.com/decred/dcrutil"
 )
 
+// API version constants
+const (
+	jsonrpcSemverString = "1.0.0"
```

Comment:
means I didn't have to write something to convert from the numbers to a version string. a function to do that already exists in this (main) package but if we refactor and split the rpc server to another package, it would require copying the function or making it available elsewhere for each package to use.  Since everything is in a single place I don't think this will cause any issues (and this is how it was specified in the source code for the dcrwallet grpc server).